kernel/kselftest.py

```diff
@@ -81,7 +84,8 @@ def test(self):
         Execute the kernel selftest
         """
         error = False
-        result = build.make(self.sourcedir, extra_args='run_tests')
+        result = build.make(
+            self.sourcedir, extra_args='%s run_tests' % self.comp)
```

Comment:
2 questions:
1. Is the null value for `self.comp` taken care ?
2. no space needed before -C ? or is that also taken care ?

Reply:
1. Yes, Line 47 handles it, and replaces with empty string.
2. Yes, as part of `build_make` it is taken care.